torchvision/io/__init__.py

```diff
@@ -15,6 +15,12 @@
     write_video,
 )
 
+from .image import (
```

Comment:
Let's revert this import and `__all__` modification

Reply:
reverted